about.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="top-page">
+        <a href=""><img src="images/facebook-icon.svg" alt="facebook"></a>
+        <a href=""><img src="images/twitter-icon.svg"></a>
+        <a href="">Logout</a>
+        <a href="">Mypage</a>
+        <a href="">Korean</a>
+    </div>
+    <header>
+        <div class="hamburger">
+            <img src="images/hamburger-icon.svg" alt="menu">
+        </div>
+        <div class="top-right">
+            <div class="mo">
+                <a href="index.html"><img src="images/main-logo.svg" alt="logo"></a>
+            </div>
+        </div>
+        <nav class="nav">
+            <a href="about.html">About</a>
+            <a href="#">Program</a>
+            <a href="#">Join</a>
+            <a href="#">Sponsor</a>
+            <a href="#">News</a>
+            <div class="campaign">
+                <a href="#">CC Campaign</a>
+            </div>
+        </nav>
+    </header>
+    <section class="home-pages">
+        <div class="home">
+            <h1>CREATIVE COMMONS</br>GLOBAL SUMMIT 2015</h1>
+        </div>
+        <div class="paragraphab">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please add correct `Horizontal line` to between of contents like [Behance Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/225585100-67bf1cd9-f506-4e0c-8aed-33061d163d58.png)